hstream/src/HStream/Server/Handler/Connector.hs

```diff
@@ -38,16 +38,16 @@ createSinkConnectorHandler
   -> ServerRequest 'Normal CreateSinkConnectorRequest Connector
   -> IO (ServerResponse 'Normal Connector)
 createSinkConnectorHandler sc
-  (ServerNormalRequest _ CreateSinkConnectorRequest{..}) = defaultExceptionHandle $ do
-  connector <- createConnector sc (TL.toStrict createSinkConnectorRequestSql)
-  returnResp connector
+  (ServerNormalRequest _ CreateSinkConnectorRequest{..}) = defaultExceptionHandle $
+    createConnector sc (TL.toStrict createSinkConnectorRequestSql)
+      >>= returnResp
 
 listConnectorsHandler
   :: ServerContext
   -> ServerRequest 'Normal ListConnectorsRequest ListConnectorsResponse
   -> IO (ServerResponse 'Normal ListConnectorsResponse)
 listConnectorsHandler ServerContext{..}
-  (ServerNormalRequest _metadata _) = defaultExceptionHandle $ do
+  (ServerNormalRequest _metadata _) = defaultExceptionHandle do
```

Comment:
Add a `$` is more readable. And if the `defaultExceptionHandle` accept a clean action(in the future), you will need `$`